uwrt_mars_rover_hw_bridge/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(uwrt_mars_rover_hw_bridge)
+find_package(catkin REQUIRED)
+catkin_package(
+#  INCLUDE_DIRS include
+#  LIBRARIES ar_tracker
```

Comment:
u need to have this uncommented and lsit the hw target here

Reply:
do you mean uncomment the LIBRARIES line? also, what do you mean by hw target?

Reply:
yes the libs line

Reply:
`uwrt_mars_rover_hw_bridge` target inside the hw repo cmakelist